core/db/migrate/20161012151358_opening_ledger_entries_for_store_credits.rb

```diff
@@ -0,0 +1,7 @@
+class OpeningLedgerEntriesForStoreCredits < ActiveRecord::Migration[5.0]
+  def change
+    Spree::StoreCredit.find_each do |store_credit|
+      Spree::StoreCreditLedgerEntry.generate_opening_ledger_entry_for(store_credit)
```

Comment:
Is it possible to do this using raw SQL or using AR models defined directly in this migration?

Doing it this way may cause issues in the future if the behaviour of `StoreCredit` or `StoreCreditLedgerEntry.generate_opening_ledger_entry_for` changes.